Resources/config/controller.xml

```diff
@@ -19,7 +19,7 @@
             <argument>%assetic.variables%</argument>
         </service>
         <service id="assetic.controller" class="%assetic.controller.class%" scope="prototype">
-            <argument type="service" id="request" />
+            <argument type="service" id="request_stack" />
```

Comment:
during the rebase the comments were lost..

@fabpot said he was not sure about BC breaks about this